--- lyx-1.4.0/lib/configure.m4.orig	2006-03-18 22:28:33.000000000 +0100
+++ lyx-1.4.0/lib/configure.m4	2006-03-18 22:29:31.000000000 +0100
@@ -238,8 +238,8 @@
   SEARCH_PROG([for the pdflatex program],latex_to_pdf,pdflatex)
 fi
 
-test $latex_to_dvi != "none" && latex_to_dvi="$latex_to_dvi \$\$i"
-test $latex_to_pdf != "none" && latex_to_pdf="$latex_to_pdf \$\$i"
+test "$latex_to_dvi" != "none" && latex_to_dvi="$latex_to_dvi \$\$i"
+test "$latex_to_pdf" != "none" && latex_to_pdf="$latex_to_pdf \$\$i"
 
 SEARCH_PROG([for a Tgif viewer and editor], TGIF_EDITOR, tgif)
 TGIF_VIEWER="$TGIF_EDITOR"
