diff -ur lyx-1.3.5.org/lib/configure lyx-1.3.5/lib/configure
--- lyx-1.3.5.org/lib/configure	2004-10-06 16:43:24.000000000 +0200
+++ lyx-1.3.5/lib/configure	2004-10-08 17:21:35.174267343 +0200
@@ -196,8 +196,8 @@
 
 fi
 
-test $latex_to_dvi != "none" && latex_to_dvi="$latex_to_dvi \$\$i"
-test $latex_to_pdf != "none" && latex_to_pdf="$latex_to_pdf \$\$i"
+test "$latex_to_dvi" != "none" && latex_to_dvi="$latex_to_dvi \$\$i"
+test "$latex_to_pdf" != "none" && latex_to_pdf="$latex_to_pdf \$\$i"
 
 # Search for an installed reLyX or a ready-to-install one
 save_PATH=${PATH}
@@ -237,7 +237,7 @@
 fi
 
 PATH=${save_PATH}
-test $tex_to_lyx_command = "reLyX" && tex_to_lyx_command="reLyX -f \$\$i"
+test "$tex_to_lyx_command" = "reLyX" && tex_to_lyx_command="reLyX -f \$\$i"
 tex_to_lyx_command=`echo $tex_to_lyx_command | sed "s,reLyX,reLyX$version_suffix,"`
 
 echo $ac_n "checking for a Noweb -> LyX converter""... $ac_c"
@@ -274,7 +274,7 @@
   literate_to_lyx_command=none
 fi
 
-test $literate_to_lyx_command = "noweb2lyx" && literate_to_lyx_command="noweb2lyx \$\$i \$\$o"
+test "$literate_to_lyx_command" = "noweb2lyx" && literate_to_lyx_command="noweb2lyx \$\$i \$\$o"
 literate_to_lyx_command=`echo $literate_to_lyx_command | sed "s,noweb2lyx,noweb2lyx$version_suffix,"`
 
 # Search something to process a literate document
@@ -312,7 +312,7 @@
   literate_to_tex_command=none
 fi
 
-test $literate_to_tex_command = "noweave" && literate_to_tex_command="noweave -delay -index \$\$i > \$\$o"
+test "$literate_to_tex_command" = "noweave" && literate_to_tex_command="noweave -delay -index \$\$i > \$\$o"
 
 echo $ac_n "checking for a HTML -> Latex converter""... $ac_c"
 echo "$ac_t""(html2latex)"
@@ -348,7 +348,7 @@
   html_to_latex_command=none
 fi
 
-test $html_to_latex_command = "html2latex" && html_to_latex_command="html2latex \$\$i"
+test "$html_to_latex_command" = "html2latex" && html_to_latex_command="html2latex \$\$i"
 
 echo $ac_n "checking for a MSWord -> Latex converter""... $ac_c"
 echo "$ac_t""(wvCleanLatex word2x)"
@@ -421,7 +421,7 @@
   image_command=none
 fi
 
-test $image_command = "convert" && image_command="convert \$\$i \$\$o"
+test "$image_command" = "convert" && image_command="convert \$\$i \$\$o"
 
 # Search for a Postscript interpreter
 echo $ac_n "checking for a Postscript interpreter""... $ac_c"
@@ -645,7 +645,7 @@
   ps_to_pdf_command=none
 fi
 
-test $ps_to_pdf_command = "ps2pdf" && ps_to_pdf_command="ps2pdf -dCompatibilityLevel=1.3 \$\$i"
+test "$ps_to_pdf_command" = "ps2pdf" && ps_to_pdf_command="ps2pdf -dCompatibilityLevel=1.3 \$\$i"
 
 # Search for a program to convert dvi to ps
 echo $ac_n "checking for a DVI to PS converter""... $ac_c"
@@ -682,7 +682,7 @@
   dvi_to_ps_command=none
 fi
 
-test $dvi_to_ps_command = "dvips" && dvi_to_ps_command="dvips -o \$\$o \$\$i"
+test "$dvi_to_ps_command" = "dvips" && dvi_to_ps_command="dvips -o \$\$o \$\$i"
 
 # Search for a program to convert dvi to pdf
 echo $ac_n "checking for a DVI to PDF converter""... $ac_c"
@@ -719,7 +719,7 @@
   dvi_to_pdf_command=none
 fi
 
-test $dvi_to_pdf_command = "dvipdfm" && dvi_to_pdf_command="dvipdfm \$\$i"
+test "$dvi_to_pdf_command" = "dvipdfm" && dvi_to_pdf_command="dvipdfm \$\$i"
 
 # We have a script to convert previewlyx to ppm
 lyxpreview_to_bitmap_command="lyxpreview2ppm.py"
@@ -944,7 +944,7 @@
   fax_command=none
 fi
 
-test $fax_command != "none" && fax_command="$fax_command \$\$i"
+test "$fax_command" != "none" && fax_command="$fax_command \$\$i"
 
 # Search for LinuxDoc support
 echo $ac_n "checking for SGML-tools 1.x (LinuxDoc)""... $ac_c"
