diff -urN lyx-1.3.1.org/lib/configure lyx-1.3.1/lib/configure
--- lyx-1.3.1.org/lib/configure	Tue Mar 25 09:22:48 2003
+++ lyx-1.3.1/lib/configure	Tue Mar 25 09:23:58 2003
@@ -188,8 +188,8 @@
 
 fi
 
-test $latex_to_dvi != "none" && latex_to_dvi="$latex_to_dvi \$\$i"
-test $latex_to_pdf != "none" && latex_to_pdf="$latex_to_pdf \$\$i"
+test "$latex_to_dvi" != "none" && latex_to_dvi="$latex_to_dvi \$\$i"
+test "$latex_to_pdf" != "none" && latex_to_pdf="$latex_to_pdf \$\$i"
 
 # Search for an installed reLyX or a ready-to-install one
 save_PATH=${PATH}
@@ -229,7 +229,7 @@
 fi
 
 PATH=${save_PATH}
-test $tex_to_lyx_command = "reLyX" && tex_to_lyx_command="reLyX -f \$\$i"
+test "$tex_to_lyx_command" = "reLyX" && tex_to_lyx_command="reLyX -f \$\$i"
 tex_to_lyx_command=`echo $tex_to_lyx_command | sed "s,reLyX,reLyX$version_suffix,"`
 
 echo $ac_n "checking for a Noweb -> LyX converter""... $ac_c"
@@ -266,7 +266,7 @@
   literate_to_lyx_command=none
 fi
 
-test $literate_to_lyx_command = "noweb2lyx" && literate_to_lyx_command="noweb2lyx \$\$i \$\$o"
+test "$literate_to_lyx_command" = "noweb2lyx" && literate_to_lyx_command="noweb2lyx \$\$i \$\$o"
 literate_to_lyx_command=`echo $literate_to_lyx_command | sed "s,noweb2lyx,noweb2lyx$version_suffix,"`
 
 # Search something to process a literate document
@@ -304,7 +304,7 @@
   literate_to_tex_command=none
 fi
 
-test $literate_to_tex_command = "noweave" && literate_to_tex_command="noweave -delay -index \$\$i > \$\$o"
+test "$literate_to_tex_command" = "noweave" && literate_to_tex_command="noweave -delay -index \$\$i > \$\$o"
 
 echo $ac_n "checking for a HTML -> Latex converter""... $ac_c"
 echo "$ac_t""(html2latex)"
@@ -340,7 +340,7 @@
   html_to_latex_command=none
 fi
 
-test $html_to_latex_command = "html2latex" && html_to_latex_command="html2latex \$\$i"
+test "$html_to_latex_command" = "html2latex" && html_to_latex_command="html2latex \$\$i"
 
 echo $ac_n "checking for a MSWord -> Latex converter""... $ac_c"
 echo "$ac_t""(wvCleanLatex word2x)"
@@ -413,7 +413,7 @@
   image_command=none
 fi
 
-test $image_command = "convert" && image_command="convert \$\$i \$\$o"
+test "$image_command" = "convert" && image_command="convert \$\$i \$\$o"
 
 # Search for a Postscript interpreter
 echo $ac_n "checking for a Postscript interpreter""... $ac_c"
@@ -632,7 +632,7 @@
   ps_to_pdf_command=none
 fi
 
-test $ps_to_pdf_command = "ps2pdf" && ps_to_pdf_command="ps2pdf -dCompatibilityLevel=1.3 \$\$i"
+test "$ps_to_pdf_command" = "ps2pdf" && ps_to_pdf_command="ps2pdf -dCompatibilityLevel=1.3 \$\$i"
 
 # Search for a program to convert dvi to ps
 echo $ac_n "checking for a DVI to PS converter""... $ac_c"
@@ -669,7 +669,7 @@
   dvi_to_ps_command=none
 fi
 
-test $dvi_to_ps_command = "dvips" && dvi_to_ps_command="dvips -o \$\$o \$\$i"
+test "$dvi_to_ps_command" = "dvips" && dvi_to_ps_command="dvips -o \$\$o \$\$i"
 
 # Search for a program to convert dvi to pdf
 echo $ac_n "checking for a DVI to PDF converter""... $ac_c"
@@ -706,7 +706,7 @@
   dvi_to_pdf_command=none
 fi
 
-test $dvi_to_pdf_command = "dvipdfm" && dvi_to_pdf_command="dvipdfm \$\$i"
+test "$dvi_to_pdf_command" = "dvipdfm" && dvi_to_pdf_command="dvipdfm \$\$i"
 
 # We have a script to convert previewlyx to ppm
 lyxpreview_to_bitmap_command="lyxpreview2bitmap.sh"
@@ -931,7 +931,7 @@
   fax_command=none
 fi
 
-test $fax_command != "none" && fax_command="$fax_command \$\$i"
+test "$fax_command" != "none" && fax_command="$fax_command \$\$i"
 
 # Search for LinuxDoc support
 echo $ac_n "checking for SGML-tools 1.x (LinuxDoc)""... $ac_c"
@@ -969,7 +969,7 @@
 fi
 
 chk_linuxdoc=no
-if test $LINUXDOC != none; then
+if test "$LINUXDOC" != none; then
   chk_linuxdoc=yes
   linuxdoc_cmd="\\def\\haslinuxdoc{yes}"
 fi
@@ -1023,7 +1023,7 @@
 fi
 
 chk_docbook=no
-if test $DOCBOOK != none; then
+if test "$DOCBOOK" != none; then
   chk_docbook=yes
   docbook_cmd="\\def\\hasdocbook{yes}"
 fi
