diff -urN lyx-1.2.0pre4.orig/config/common.am lyx-1.2.0pre4/config/common.am
--- lyx-1.2.0pre4.orig/config/common.am	Fri May  3 01:09:50 2002
+++ lyx-1.2.0pre4/config/common.am	Fri May  3 01:10:05 2002
@@ -9,4 +9,4 @@
 BOOST_INCLUDES = -I$(top_srcdir)/boost
 SIGC_INCLUDES = -I$(top_srcdir) -I$(top_builddir)
 
-LIBS =
+
diff -urN lyx-1.2.0pre4.orig/src/Makefile.am lyx-1.2.0pre4/src/Makefile.am
--- lyx-1.2.0pre4.orig/src/Makefile.am	Fri May  3 01:09:49 2002
+++ lyx-1.2.0pre4/src/Makefile.am	Fri May  3 01:10:38 2002
@@ -6,7 +6,7 @@
 
 SUBDIRS = mathed insets graphics support frontends
 
-LIBS += @LIBS@
+LIBS = @LIBS@
 
 bin_PROGRAMS = lyx
 
diff -urN lyx-1.2.0pre4.orig/src/frontends/Makefile.am lyx-1.2.0pre4/src/frontends/Makefile.am
--- lyx-1.2.0pre4.orig/src/frontends/Makefile.am	Fri May  3 01:09:49 2002
+++ lyx-1.2.0pre4/src/frontends/Makefile.am	Fri May  3 01:10:48 2002
@@ -6,7 +6,7 @@
 
 noinst_LTLIBRARIES = libfrontends.la
 
-LIBS += @FRONTEND_LDFLAGS@ @FRONTEND_LIBS@
+LIBS = @FRONTEND_LDFLAGS@ @FRONTEND_LIBS@
 
 INCLUDES = $(FRONTEND_INCLUDES) -I$(srcdir)/..  $(SIGC_CFLAGS) $(BOOST_INCLUDES)
 libfrontends_la_LIBADD = @FRONTEND_GUILIB@ controllers/*.lo

