--- lyx-1.3.7/configure.ac.orig	2006-02-01 13:38:30.000000000 +0100
+++ lyx-1.3.7/configure.ac	2006-02-01 13:39:03.000000000 +0100
@@ -20,7 +20,7 @@
 
 # The list of languages known to LyX (actually we only include the languages 
 # that are complete enough to be useful). This is needed by GNU gettext
-ALL_LINGUAS="da de es eu fi fr hu it nl nn no pl ro ru sk sl tr"
+ALL_LINGUAS="da de es eu fi fr hu it nb nl nn pl ro ru sk sl tr"
 
 ### Set the execute permissions of the various scripts correctly
 for file in config/install-sh config/mkinstalldirs lib/configure ; do
--- lyx-1.3.7/config/configure.ac.orig	2006-02-01 13:41:51.000000000 +0100
+++ lyx-1.3.7/config/configure.ac	2006-02-01 13:42:13.000000000 +0100
@@ -20,7 +20,7 @@
 
 # The list of languages known to LyX (actually we only include the languages 
 # that are complete enough to be useful). This is needed by GNU gettext
-ALL_LINGUAS="da de es eu fi fr hu it nl nn no pl ro ru sk sl tr"
+ALL_LINGUAS="da de es eu fi fr hu it nl nb nn pl ro ru sk sl tr"
 
 ### Set the execute permissions of the various scripts correctly
 for file in config/install-sh config/mkinstalldirs lib/configure ; do
--- lyx-1.3.7/config/configure.in.orig	2006-02-01 13:41:55.000000000 +0100
+++ lyx-1.3.7/config/configure.in	2006-02-01 13:42:24.000000000 +0100
@@ -20,7 +20,7 @@
 
 # The list of languages known to LyX (actually we only include the languages 
 # that are complete enough to be useful). This is needed by GNU gettext
-ALL_LINGUAS="da de es eu fi fr hu it nl nn no pl ro ru sk sl tr"
+ALL_LINGUAS="da de es eu fi fr hu it nl nb nn pl ro ru sk sl tr"
 
 ### Set the execute permissions of the various scripts correctly
 for file in config/install-sh config/mkinstalldirs lib/configure ; do
