--- configure.in.orig	Wed Apr 24 17:43:48 2002
+++ configure.in	Fri May  3 10:41:43 2002
@@ -193,7 +193,7 @@
 	FRONTEND_INFO=`cat <<EOF
     libXpm version:               ${XPM_VERSION}\n\
     libforms version:             ${XFORMS_VERSION}\n
-`
+EOF`
 	;;
   gnome)
     LYX_PATH_XPM
@@ -347,7 +347,7 @@
 ${FRONTEND_INFO}\
   LyX binary dir:                 ${real_bindir}\n\
   LyX files dir:                  ${real_datadir}\n
-`
+EOF`
 
 AC_SUBST(VERSION_INFO)
 
--- acinclude.m4~	Wed Apr 24 17:49:10 2002
+++ acinclude.m4	Fri May  3 11:45:09 2002
@@ -81,9 +81,7 @@
 **** (see the section 'Problems' in the INSTALL file)
 $lyx_error_txt
 $lyx_warning_txt
-deleting cache $cache_file
 EOF
-  rm -f $cache_file
 else
 
 if test x$lyx_warning = xyes; then
--- configure.in~	Mon Jun 10 17:28:30 2002
+++ configure.in	Mon Jun 10 17:35:50 2002
@@ -66,13 +66,6 @@
 LYX_USE_FRONTEND
 dnl The code below is not in a macro, because this would cause big
 dnl problems with the AC_REQUIRE contained in KDE_DO_IT_ALL.
-case "$lyx_use_frontend" in
-  gnome)
-    GNOME_INIT
-    GNOME_COMPILE_WARNINGS
-    GNOME_X_CHECKS
-esac
-
 ### Check for a C++ compiler
 LYX_PROG_CXX
 ### Some checks on what the C++ compiler can(not) do
